libbeat/docs/http-endpoint.asciidoc

```diff
@@ -0,0 +1,149 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/http-endpoint.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+[[http-endpoint]]
+== HTTP Endpoint
+
+experimental[]
+
+{beatname_uc} can expose internal metrics through a HTTP endpoint. For security
+reasons the endpoint is disabled by default.
+
+The HTTP endpoint has the following configuration settings:
+
+`http.enabled`:: (Optional) Enable the http endpoint. Default is `false`.
+`http.host`:: (Optional) Bind to this hostname or IP address.
+It is recommended to use only localhost. Default is `localhost`
+`http.port`:: (Optional) Port on which the HTTP endpoint will bind. Default is `5066`.
+
+[float]
+=== Paths
+
+This is the list of paths you can access. For pretty JSON output append ?pretty to the URL.
+
+
+`/` provides basic info from the {beatname_uc}. Example:
```

Comment:
Maybe provide an example curl command to show this (like we do in the overview topic in the LS Reference).